airflow-core/src/airflow/ui/public/i18n/locales/en/dags.json

```diff
@@ -13,7 +13,8 @@
       "active": "Active",
       "all": "All",
       "paused": "Paused"
-    }
+    },
+    "runIdPatternFilter": "Add run_id filter"
```

Comment:
Can we add a tooltip or similar that helps the user to know that "%" is the wildcard? Else user probably does not know or attempts to use "*".

Reply:
That would also be great. So user enters "abc" and the query is automatically be made with "%abc%"